src/graph_1.py

```diff
@@ -13,7 +13,7 @@ def nodes(self):
         """Return a list of all nodes in the graph."""
         return self.node_list
 
-    def edges(self):
+    def edge_report(self):
         """Return a list of all edges in the graph."""
         return list(self.edges.key())
```

Comment:
The method shouldn't be called `edge_report`. Also, this isn't getting me a list of all the edges. This is getting me a list of all those nodes that have an outbound edge.

Reply:
NVM I see how you're handling edges now.